PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/MetaDataTest.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.pl2016gr2.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class MetaDataTest {
+  
+  private MetaData md;
```

Comment:
It is indeed longer, but not that long. `md` doesn't describe the variable in any meaningful way at all and with the easy of use of code completion in IDEs it doesn't even take longer to "type" a 30 character variable name than a 5 character variable name. Using very short variable names in general is in my opinion a bad practise and not a good argument to use short variable names.